### PR TITLE
Handle jenkins.baseline on AddPluginsPom

### PR DESCRIPTION
**Pull request by jonesbusy**
## What's changed?

Handle presence of `jenkins.baseline` on artifact ID. 

Doesn't perform migration by adding the `jenkins.baseline` property which should be done at some point

## What's your motivation?

Jenkins changed their archetype to include the `jenkins.baseline` property

### Checklist
- [ ] I've added unit tests to cover both positive and negative cases
- [ ] I've read and applied the [recipe conventions and best practices](https://docs.openrewrite.org/authoring-recipes/recipe-conventions-and-best-practices)
- [ ] I've used the IntelliJ IDEA auto-formatter on affected files

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -1,3 +1,10 @@
+configurations.all {
```
Comment (jonesbusy) [inline build.gradle.kts:1]: No idea what's going on here, but using latest SNAPSHOT cause issues

See (https://rewriteoss.slack.com/archives/C01A843MWG5/p1730980508003589)

Comment (timtebeek) [reply build.gradle.kts:1]: Thanks for pointing these out; fixed in
- https://github.com/openrewrite/rewrite-jenkins/pull/78

**Comment by jonesbusy** [comment]
Thanks for the help @timtebeek this is now ready for rewiew

**Comment by jonesbusy** [comment]
FYI @gounthar

**Comment by gounthar** [comment]
Thanks!

**Review by timtebeek — APPROVED** [review]
Thanks @jonesbusy !

**Pull request merged**